init-asgs.sh

```diff
@@ -164,9 +165,9 @@ case "$platform" in
        export PLATFORM_INIT=$(readlink -f "${_PLATFORM_INIT[$platform]}")
        source "$PLATFORM_INIT"
      else
-      echo "Can't find 'init.sh' for '$platform' or Unknown defaults for platform '$platform', using "$HOME" as 'WORK' and 'SCRATCH' directories..."
-      WORK=${WORK:-$HOME}
-      SCRATCH=${SCRATCH:-$HOME}
+      echo "Can't find 'init.sh' for '$platform' or Unknown defaults for platform '$platform', using "$ASGS_HOME" as 'WORK' and 'SCRATCH' directories..."
+      WORK=${WORK:-$ASGS_HOME}
+      SCRATCH=${SCRATCH:-$ASGS_HOME}
```

Comment:
this may the issue with `$RUNDIR` ending up as `$ASGS_HOME` as we discussed

Reply:
I think it's an artifact on my particular installation. I was able to show that it sets to what is exported based on the platform specific stuff that's set in `init-asgs.sh` or (preferably) in the platform's definition, like in `./platforms/mike/init.sh`. We'll just have to keep an eye on it.